.golangci.yml

```diff
@@ -0,0 +1,235 @@
+# options for analysis running
+run:
+  # default concurrency is a available CPU number
+  concurrency: 4
+
+  # timeout for analysis, e.g. 30s, 5m, default is 1m
+  deadline: 5m
+
+  # exit code when at least one issue was found, default is 1
+  issues-exit-code: 1
+
+  # include test files or not, default is true
+  tests: false
```

Comment:
We would need to enable this in the next pr and fix lint in tests.

Reply:
agreed, I will raise one issue for this once this PR is merged.